level/level.c

```diff
@@ -240,8 +240,16 @@ level* make_level(long int map_seed) {
     push_inventory(lvl->player, stick);
 
     for (int i=0; i < lvl->mob_count-1; i++) {
-        lvl->mobs[i]->x = rand_int(level_width);
-        lvl->mobs[i]->y = rand_int(level_height);
+        int x = 0;
+        int y = 0;
+
+        while (lvl->tiles[x][y] != TILE_FLOOR) {
```

Comment:
There must be a more elegant way to do this, but I don't think I mind this.

Reply:
Nope, that's about it. Short of changing how we store the level structure.

Reply:
My only suggestion would be to start with a random location not 0,0.

Reply:
You are probably right. I wanted to avoid having the same code in two places so I chose (0,0) as a known wall so the while() always fires off. It's goofy but I like not duplicating the rand_int() calls for who knows what reason. Gonna leave it, but maybe it changes later.